spirv/release.toml

```diff
@@ -7,5 +7,7 @@ sign-tag = true
 
 pre-release-replacements = [
   {file="README.md", search="spirv = .*", replace="{{crate_name}} = \"{{version}}\""},
-  {file="../rspirv/Cargo.toml", search="spirv = .*", replace="{{crate_name}} = \"={{version}}\""},
-]
\ No newline at end of file
+  {file="../rspirv/Cargo.toml", search="spirv = \\{ version = \"=.*\", path = \"../spirv\" \\}", replace="{{crate_name}} = { version = \"={{version}}\", path = \"../spirv\" }" },
```

Comment:
Not sure if we should keep the `=` sign; it means that `rspirv` is fixed to spirv `0.1.0` exactly. If we ever release non-breaking spirv changes in `0.1.1` (which is what we want to do now with the semver restart) none of the `rspirv` crates will be able to use it unless they get a bump too.


(I initially added the `=` but don't recall why, since as explained above it obviously doesn't make much sense).

Reply:
I'll remove it.